ts/Core/Series/DataLabel.ts

```diff
@@ -763,6 +763,13 @@ namespace DataLabel {
                                 dataLabel.css(style).shadow(
                                     labelOptions.shadow
                                 );
+                            } else if (style.width) { // #20499
+                                // In styled mode with a width property set,
+                                // the width should be applied to the
+                                // dataLabel. (#20499)
+                                dataLabel.css({
+                                    width: style.width
```

Comment:
Now that we allowed width, I think we also should allow `textOverflow`. Otherwise implementers will not be able to apply ellipsis. It can't be set in CSS alone because it requires a JavaScript interpretation for SVG text.

```suggestion
                                    width: style.width,
                                    textOverflow: style.textOverflow
```